packages/website/src/model/page.ts

```diff
@@ -0,0 +1,23 @@
+import titleCase from 'title-case';
+import { DocsPage, Page } from '../../types';
+/* eslint-disable import/prefer-default-export */
+/**
+ * Business logic related to pages
+ * @file page.ts
+ */
+
+/**
+ * Whether the page is a docs page. Currently just used as a typeguard so that we can access
+ * meta when it does not exist on all types of pages.
+ */
+const isDocsPage = (page: any): page is DocsPage => {
+  return page.meta != null;
```

Comment:
So, this equality check works will return false if page meta is null OR undefined right?

What's the reason we're using this? (I'm always wary whenever we start using non triple compares)

Reply:
Yep, we're effectively making sure that `page.meta` is defined. `!= null` is essentially a shorthand for `!== null && typeof !== 'undefined'`. Imo it's one of the few valid use cases of loose comparison.

Reply:
We should add the eslint rule to enforce always triple ===